MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -65,18 +65,21 @@ open class BaseInstructionsBannerView: UIControl {
     }
     
     @objc func draggedInstructionsBanner(_ sender: Any) {
-        if let gestureRecognizer = sender as? UIPanGestureRecognizer, gestureRecognizer.state == .ended {
+        if let gestureRecognizer = sender as? UIPanGestureRecognizer, gestureRecognizer.state == .ended, let delegate = delegate {
```

Comment:
This prevents the two lines below from running if the delegate isn’t set. Therefore, if the delegate isn’t set, then the indicator view doesn’t get hidden.

Reply:
Yeah I guess that's the point though. We should only toggle the indicator if the developer implements the delegate methods.